dotcom-rendering/src/layouts/InteractiveLayout.tsx

```diff
@@ -238,6 +239,13 @@ export const InteractiveLayout = (props: WebProps | AppsProps) => {
 	const renderAds = isWeb && canRenderAds(article);
 	return (
 		<>
+			{isApps && (
+				<>
+					<Island priority="critical">
+						<NativePlatformWrapper />
+					</Island>
+				</>
```

Comment:
Is this fragment redundant?

```suggestion
			<Island priority="critical">
				<NativePlatformWrapper />
			</Island>
```

Reply:
Ah sorry, thought there might be some sort of convention based on https://github.com/guardian/dotcom-rendering/blob/81c50181c83ac38927d08e41071ebf2e5c5b2caa/dotcom-rendering/src/layouts/InteractiveLayout.tsx#L308-L312, have removed

Reply:
Not at all, this is our bad as the linting rules should catch this. Most likely a hangover from a refactor. Thanks for the pointer.